React/my-app/src/redux/accounts/actions.js

```diff
@@ -35,11 +40,13 @@ export const registerNewUser = body => {
           type: actions.NEW_USER_SUCCESS,
           payload: { error: 0 }
```

Comment:
don need thats the payload error on the success. Just modifique this when enter on reducer action